app/javascript/components/pages/dashboard/utils/to-reporting-location-table.js

```diff
@@ -7,7 +7,7 @@ import { INDICATOR_NAMES } from "../constants";
 const reportingLocationLabel = (reportingLocationConfig, i18n) => {
   const locationTypes = [];
 
-  reportingLocationConfig.label_keys.forEach(key => {
+  reportingLocationConfig?.label_keys?.forEach(key => {
```

Comment:
Why do you need this change?

Reply:
For null checking

Reply:
Yes, but why this was not needed before?

Reply:
To test if object is undefined & has any properties that matches the keys